docs/en/advanced-concepts/hybrid.md

```diff
@@ -138,7 +138,7 @@ public function testThings()
 
 ### Automating hybrid Android apps
 
-Appium comes with built-in hybrid support via Chromedriver. Appium also uses
+Appium comes with built-in hybrid support via [Chromedriver](/docs/en/advanced-concepts/chromedriver.md). Appium also uses
```

Comment:
I would change this link so that it includes the previous part of the sentence, i.e. "[built-in hybrid support via Chromedriver](/docs/en/advanced-concepts/chromedriver.md)"